tasks/install-server.yml

```diff
@@ -58,6 +59,9 @@
     owner: "root"
     group: "root"
   when: ansible_service_mgr == "systemd"
+  notify:
+    - reload minio systemd
```

Comment:
Why not just add `daemon_reload: true` to the restart handler? This is how [we handle this](https://github.com/cloudalchemy/ansible-prometheus/blob/master/handlers/main.yml).

Reply:
because it is a service (generic init.d/systemd) module, not the systemd module. :wink:

Reply:
Ahh, right, we do systemd-only in our roles. :-)